ticdc/ticdc-sink-to-mysql.md

```diff
@@ -90,9 +90,13 @@ TiCDC replication lag increases in the following scenarios:
 - Time-consuming DDL statements, such as `add index`, are executed in the upstream.
 - The PD is configured with aggressive scheduling strategies, resulting in frequent transfer of Region leaders, or frequent Region merge or Region split.
 
+> **Note:**
+>
+> Starting from v6.1.1, the eventually consistent replication feature of TiCDC supports Amazon S3-compatible object storage. Starting from v6.1.4, this feature supports GCS- and Azure-compatible object storage.
+
 ### Prerequisites
 
-- Prepare a highly available Amazon S3 storage or NFS system for storing TiCDC's real-time incremental data backup files. These files can be accessed in case of an primary cluster disaster.
+- Prepare a highly available object storage or NFS system for storing TiCDC's real-time incremental data backup files. These files can be accessed in case of a disaster in the upstream.
```

Comment:
```suggestion
- Prepare a highly available object storage or NFS for storing TiCDC's real-time incremental data backup files. These files can be accessed in case of a disaster in the upstream.
```